common/saber_base.h

```diff
@@ -76,6 +76,16 @@ DEFINE_ALL_EFFECTS();
 constexpr size_t NUMBER_OF_EFFECTS = (size_t)EffectTypeHelper::NUMBER_OF_EFFECTS;
 #undef DEFINE_EFFECT
 
+#ifdef ENABLE_ALL_MENU_OPTIONS
```

Comment:
Can't this go in your prop file?

Reply:
I had tried but the DYNAMIC_BLADE_DIMMING code below was missing causing compile error, it fixed when I moved to here.

Reply:
You just need to put it before the #include "prop_base.h" line.

Reply:
OK, I thought I tried that but I'll do it again and re-test tomorrow.

Reply:
removed